src/components/catalog/CatalogCard.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import '../../css/catalog/CatalogCard.css';
+
+export default class CatalogCard extends Component {
+  constructor(props) {
+    super(props);
+
+    this.modalContainer = React.createRef();
+
+    this.state = {
+      course: {
+        title: '',
+        completedCourses: '',
+        totalCourses: '',
+        img: '',
+      },
+    };
+  }
+  static propTypes = {
+    course: PropTypes.object.isRequired,
+    clickCourse: PropTypes.func.isRequired,
+  };
+
+  // ===============================================================
+  // Handlers
+    /**
+     * TODO: Handle route when course is clicked
+     * @param {Object} selectedCourse the object containing the data for the course to be displayed
+     */
+  clickCourse = selectedCourse => {
+    console.log(selectedCourse);
+  };
+  
+  componentDidMount(){
+    this.setState({
+      course: {
+        title: this.props.course.title,
+        completedCourses: this.props.course.completedCourses,
+        totalCourses: this.props.course.totalCourses,
+        img: this.props.course.img,
+      },
+    });
+  }
+
+  // ===============================================================
+  // Render
+
+  render() {
+    const { course, clickCourse } = this.props;
+
+    return (
+      <div
+        id="course-item"
```

Comment:
This is a repeated modular component, so this can't be an `id`, since this component will not be unique.

```suggestion
        className="course-item"
```